app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1804,7 +1804,7 @@ table.orders{
 
 .order-details {
   border: 1px solid red;
-  background-color: #6E2602; 
+  background-color: #6E2602;
```

Comment:
Color literals like `#6E2602` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#6E2602` should be written as `#6e2602`